SupportScripts/Python/DownloadGroupMedia/download_group_media.py

```diff
@@ -0,0 +1,75 @@
+import os
+import requests
+from urllib.parse import urlparse
+
+ACCESS_TOKEN = 'YOUR_ACCESS_TOKEN'
+GROUP_ID = 'WORPLACE_GROUP_ID'
```

Comment:
Hi @b-bell , thanks for the contrib.  Can I suggest you help start the move to [a more secure pattern of reading from ENV vars.](https://github.com/fbsamples/workplace-platform-samples/pull/141/files#diff-5ae1da38b781f288d78d7593643103181da9f267ace489c0e9167e395f1f95b5R215).  Not every user may be savvy enough to set them, so the fallback is there, too.  But ideally we can help some people keep their env secrets safer.